services/management/grpc/actions_server.go

```diff
@@ -294,6 +295,73 @@ func (s *actionsServer) StartPTSummaryAction(ctx context.Context, req *managemen
 	}, nil
 }
 
+func pointerToAgentType(agentType models.AgentType) *models.AgentType {
+	return &agentType
+}
+
+// TBD: This prototype is here to allow compilation. It will be replaced by PMM-4172 content when its PR approved.
+// StartPTMySQLSummaryAction starts pt-mysql-summary action.
+//nolint:lll
+func (s *actionsServer) StartPTMySQLSummaryAction(context.Context, *managementpb.StartPTMySQLSummaryActionRequest) (*managementpb.StartPTMySQLSummaryActionResponse, error) {
+	panic("not implemented yet")
+}
+
+// StartPTPgSQLSummaryAction starts pt-pg-summary (PostgreSQL) action and returns the pointer to the response message
+//nolint:lll
+func (s *actionsServer) StartPTPgSQLSummaryAction(ctx context.Context, req *managementpb.StartPTPgSQLSummaryActionRequest) (*managementpb.StartPTPgSQLSummaryActionResponse, error) {
+	// Need to get the service id's pointer to retrieve the list of agent pointers therefrom
+	// to get the particular agentID from the request.
+	service, err := models.FindServiceByID(s.db.Querier, req.ServiceId)
+	if err != nil {
+		return nil, err
+	}
+
+	pmmAgents, err := models.FindPMMAgentsRunningOnNode(s.db.Querier, service.NodeID)
+	if err != nil {
+		return nil, status.Errorf(codes.NotFound, "No pmm-agent running on this node")
+	}
+
+	pmmAgentID, err := models.FindPmmAgentIDToRunAction(req.PmmAgentId, pmmAgents)
+	if err != nil {
+		return nil, err
+	}
+
+	res, err := models.CreateActionResult(s.db.Querier, pmmAgentID)
+	if err != nil {
+		return nil, err
+	}
+
+	// Exporters to be filtered by service ID and agent type
+	agentFilter := models.AgentFilters{PMMAgentID: "", NodeID: "",
+		ServiceID: req.ServiceId, AgentType: pointerToAgentType(models.PostgresExporterType)}
+
+	// Need to get the postgress exporters to get the username and password therefrom
+	pExporters, err := models.FindAgents(s.db.Querier, agentFilter)
```

Comment:
https://github.com/percona/pmm-managed/pull/510#discussion_r523804000

Reply:
p as postgres. Do you want me to type there entire word "postgres"?

Reply:
We want two things.

First, we want you to be consistent. The other PR introduces a very similar function where this variable is called `mysqldExporters`. From the typos in the error message, it looks like you copied that code (which is a good thing there) but then decided to use your preferred style of naming variables again. Please be consistent with other names in this file. Please call it `postgresExporter`.

Second, we want it to be the last time we discuss that. That other comment was created more than three weeks ago. We discuss consistency for _over a month_, yet you are consistently inconsistent with existing code and consistently apply your own preferences, ignoring pre-existing project style. Those daily back-and-forths are tiring for everyone. They have to stop.

Reply:
The name was not according to my preference at all. My preference is to see maximum from the variable's name without necessity of clicking it and investigating around. Here p was an abbreviation of postgreSql in English form combination: an adjective followed by a noun.
I could've chosen one of these: p, pg, pgSql, postgreSql, postgres, or postgre? I've chosen p. The simplest one. Today I learnt from Nurlan you use 3 names for postgreSql. If one sees it in so many forms, he will not most likely think that you have predefined just 3 possible combinations.
I will call it postgresExporters. This was the only thing I asked.